gemd/units/citrine_en.txt

```diff
@@ -180,12 +181,12 @@ mil = π / 32000 * radian = Mil
 steradian = radian ** 2 = sr = Steradian
 square_degree = (π / 180) ** 2 * sr = sq_deg = sqdeg = Square_Degree
 
-# Logarithmic ratio
-bel = 0.5 * ln10 * neper = Bel
-
 # Information
-byte = 8 * bit = B = octet = Byte = Octet
-baud = bit / second = Bd = bps = Baud = BPS
```

Comment:
Why did we drop the capitalized forms?

Reply:
Good catch.  The capitalized forms are all changes that we made, but when they moved these to a new section, I'd missed that edit.